models/hourly_spend.sql

```diff
@@ -31,6 +31,13 @@ hours as (
     from hour_spine
 ),
 
+usage_in_currency_daily as (
+    select
+        * exclude(usage_type),
+        replace(usage_type, 'overage-', '') as usage_type
+    from {{ ref('stg_usage_in_currency_daily') }}
+),
```

Comment:
i think it will now be possible to have two entries on the same day for a given usage type:

`overage-compute` and `compute` will get turned into `compute` and `compute`

checking if this is ok?

Reply:
yeah looking further i think we will need to do something like `group by day, usage type` and aggregate spend

Reply:
Very good catch, thanks. I haven't thought about it. In the scope of this package, this model is only used by `daily_spend` ([source](https://get-select.github.io/dbt-snowflake-monitoring/#!/model/model.dbt_snowflake_monitoring.hourly_spend?g_v=1&g_i=%2Bhourly_spend%2B)), which will do a group by and correctly flatten multiple entries into 1, but given the purpose of the model, I think it feels confusing if it returns multiple rows for the same service and hour. I've updated the PR to group those possible multiple rows in the CTE.

Reply:
Good point that it will get handled naturally, but to avoid confusion i think it makes sense to group. thanks for correcting